server/src/main/java/io/druid/server/http/CoordinatorDynamicConfigsResource.java

```diff
@@ -56,14 +65,36 @@ public Response getDynamicConfigs()
     ).build();
   }
 
+  // default value is used for backwards compatibility
   @POST
   @Consumes(MediaType.APPLICATION_JSON)
-  public Response setDynamicConfigs(final CoordinatorDynamicConfig dynamicConfig)
+  public Response setDynamicConfigs(final CoordinatorDynamicConfig dynamicConfig,
+                                    @HeaderParam("author") @DefaultValue("") final String author,
```

Comment:
same here, we should probably also define constants somewhere for the "X-Druid-Author" and "X-Druid-Comment" headers